kfs-purap/src/main/java/edu/arizona/kfs/module/purap/document/web/struts/PaymentRequestAction.java

```diff
@@ -53,7 +53,7 @@ private static ConfigurationService getConfigurationService() {
         return configurationService;
     }
 
-    @Override
+

```

Comment:
This was removed because the import for the PaymentRequestDocument (on the method signature) was changed to edu.arizona.kfs.module.purap.document.PaymentRequestDocument. This changed the method signature such that Eclipse and Maven view the @Override annotation as an error. Changing the import was necessary for loadIncomeTypesFromAccountLines below. If you want, I can remove the import change and specifically call out the edu version of the document in the method, however if this class is changed in the future for additional mods, and these changes need access to the edu version of the Document class, those methods too will need to specifically call the edu Document class.